mutiny-core/src/auth.rs

```diff
@@ -224,10 +225,8 @@ mod test {
 
     use super::*;
 
-    async fn create_manager(db_prefix: String) -> AuthManager {
-        let storage = MutinyStorage::new("".to_string(), Some(db_prefix))
-            .await
-            .unwrap();
+    fn create_manager() -> AuthManager<MemoryStorage> {
+        let storage = MemoryStorage::default();
```

Comment:
Yeah we could, the goal however is to have the storage trait be our source of truth and as long as things work against it, then mutiny works. It would be a good idea to replicate all the tests in mutiny-wasm tho, just to be sure.